### PR TITLE
eventSchemas-jsonSchemas

### PR DESCRIPTION
**Pull request by bipin-rudder**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: event-schema/event_schema_api.go [hunk 1]
```diff
@@ -54,6 +56,181 @@ func (manager *EventSchemaManagerT) GetEventModels(w http.ResponseWriter, r *htt
...
+	for _, eventModel := range eventModels {
+		flattenedSchema := make(map[string]interface{})
+		err := json.Unmarshal(eventModel.Schema, &flattenedSchema)
+		if err!=nil {
+		pkgLogger.Errorf("Unable to unmarshal eventModelSch")
+		}
+		unFlattened, err := unflatten(flattenedSchema)
+		if err!= nil {
+		}
+		filtered, ok := unFlattened["properties"].(map[string]interface{})
```
Comment (psrikanth88) [inline event-schema/event_schema_api.go:109]: are we generating schemas only for track events? what about identify etc.

**Review by psrikanth88 — APPROVED** [review]
https://github.com/rudderlabs/rudder-server/pull/1170/files#r690216956

File: event-schema/event_schema_api.go (hunk 1 above)
Comment (achettyiitr) [reply event-schema/event_schema_api.go]: Done.

**Comment by achettyiitr** [comment]
Created a new Pr for from this.

**Pull request closed without merging**